### PR TITLE
Update special-character-restrictions-in-policy-names.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/special-character-restrictions-in-policy-names.md [hunk 1]
```diff
@@ -37,9 +37,12 @@ If you have a policy with special characters, you will need to either edit the p
 ## To remove special characters
 
 **Step 1 - Make a remote connection with PowerShell.**
-[Set up your computer for Windows PowerShell](https://docs.microsoft.com/skypeforbusiness/set-up-your-computer-for-windows-powershell/set-up-your-computer-for-windows-powershell) if you haven't yet.
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/special-character-restrictions-in-policy-names.md:41]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**